src/js/components/search/topFilterBar/filterGroups/KeywordFilterGroup.jsx

```diff
@@ -19,22 +19,37 @@ export default class KeywordFilterGroup extends React.Component {
         super(props);
 
         this.removeFilter = this.removeFilter.bind(this);
+        this.clearGroup = this.clearGroup.bind(this);
     }
 
-    removeFilter() {
+    removeFilter(value) {
     // remove a single filter item
+        const newValue = this.props.redux.reduxFilters.keyword.delete(value);
+        this.props.redux.updateGenericFilter({
+            type: 'keyword',
+            value: newValue
+        });
+    }
+
+    clearGroup() {
         this.props.redux.clearFilterType('keyword');
     }
 
     generateTags() {
     // check to see if a keyword is provided
-        const tags = this.props.filter.values.map((value) => (
-            {
+        const tags = [];
+
+        const keywords = this.props.filter.values;
+
+        keywords.forEach((value) => {
+            const tag = {
                 value: `${value}`,
-                title: `${value}`,
+                title: `KEYWORD | ${value}`,
```

Comment:
Was this change in a mock?  Or on Slack?  I don't see adding the text "Keyword" to the filter chips in the AC.

Reply:
ugh. This was in a test I was doing. I can remove it. I thought I got all unused code